mifosng-android/src/main/java/com/mifos/api/services/DocumentService.java

```diff
@@ -98,10 +98,10 @@
      */
     @PUT("{entityType}/{entityId}/" + APIEndPoint.DOCUMENTS + "/{documentId}")
     @Multipart
-    Observable<GenericResponse> updateDocument(@Path("entityType") String entityType,
-                                               @Path("entityId") int entityId,
-                                               @Path("documentId") int documentId,
-                                               @Part("name") String nameOfDocument,
-                                               @Part("description") String description,
-                                               @Part() MultipartBody.Part typedFile);
+    Observable<DocumentRelatedResponse> updateDocument(@Path("entityType") String entityType,
```

Comment:
@tarun0 When we know what will be response for many requests the It's better to follow genericity and Please use `GenericResponse`